src/Files.App/ViewModels/Settings/AppearanceViewModel.cs

```diff
@@ -129,5 +145,19 @@ public string AppThemeBackgroundColor
 				}
 			}
 		}
+
+		private string selectedSystemBackdrop;
+		public string SelectedSystemBackdrop
```

Comment:
```suggestion
		public string SelectedBackdropMaterial
```
Can you do this for the other variable names as well?